### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by mcauley-penney** [Empyrean-Capstone/Empyrean#4]
1. Discuss with Dr. Llama what license he feels fits the project best. Typical choices would be GPL3 or MIT

2. Choose it from the `Add a license` interface so that [Licensee](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license) can interpret it and make it look nice

**Comment by mcauley-penney** [comment]
Summary: We *should* be okay to use a permissive license. MIT should be fine.

Frontend: Using `npx license-checker`, I discovered that there was only one repository, node-forgev1.3.1, that *potentially* used a copyleft license (see the image below for a summary). I didn't realize that the "or" in that description meant that it was up to the repo that depended on it, so we can choose the more permissive license.

![image](https://user-images.githubusercontent.com/59481467/230739594-2fa1ed73-9668-4efe-8d56-225aea30808e.png)


Backend: All dependencies fall under LGPL or more permissive, that I found.
